data.js

```diff
@@ -56,3 +56,122 @@ const usersArray = [
     gender: 'female'
   }
 ];
+
+
+// Iteration 1: for...of loop
+const getFirstNames = arr => {
+  const userFirstNames = [];
+  for(let user of arr){
+    let firstName = user.firstName;
```

Comment:
for cleaner code and less repetition you could have destructured the first name directly
```js
let {firstName} = user
```

You are not repeating the firstName unnecessarily.